pkgs/servers/sql/postgresql/default.nix

```diff
@@ -87,6 +98,10 @@ let
       ./patches/hardcode-pgxs-path.patch
       ./patches/specify_pkglibdir_at_runtime.patch
       ./patches/findstring.patch
+
+      # Remove references to output paths from configure flags
+      # recorded in the software.
+      ./patches/remove-refs-from-configure-flags.patch
```

Comment:
instead of patching `configure.ac`, can't we patch `configure` or maybe `confdefs.h` in `postConfigure`?